src/main/java/ch/cern/eam/wshub/core/services/workorders/WorkOrderMiscService.java

```diff
@@ -32,4 +33,7 @@
 
     @Operation(logOperation = INFOR_OPERATION.TASKPLAN_C)
     String createTaskPlan(InforContext context, TaskPlan taskPlan) throws InforException;
+
+    @Operation(logOperation = INFOR_OPERATION.WO_EQP_LINEAR_DETAILS_R)
+    AdditionalWOEquipDetails getWOEquipLinearDetails(InforContext context, String eqCode) throws InforException;
```

Comment:
Since we are already in the context of the WorkORderMiscService, could we name this method something like 'getLinearEquipmentDetails'?

Reply:
Yes, updated